src/libraries/System.Text.Json/src/PACKAGE.md

```diff
@@ -1,11 +1,258 @@
+Console.WriteLine($"firstDate={ firstDate}");
+//output:
+//firstDate = "2019-08-01T00:00:00"
+```
+
+Using the reader/writer
+```csharp
+/*
+  SharpLab tools in Run mode:
+    • value.Inspect()
+    • Inspect.Heap(object)
+    • Inspect.Stack(value)
+    • Inspect.MemoryGraph(value1, value2, …)
+*/
+using System;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+var writerOptions = new JsonWriterOptions
+{
+    Indented = true
+};
+
+using var stream = new MemoryStream();
+using var writer = new Utf8JsonWriter(stream, writerOptions);
+
+writer.WriteStartObject();
+writer.WriteString("date", DateTimeOffset.Parse("8/2/2023 9:00 AM"));
+writer.WriteNumber("temp", 42);
+writer.WriteEndObject();
+writer.Flush();
+
+var jsonBytes = stream.ToArray();
+string json = Encoding.UTF8.GetString(jsonBytes);
+Console.WriteLine(json);
+// {
+//   "date": "2023-08-02T09:00:00+00:00"
+//   "temp": 42
+// }
+
+var readerOptions = new JsonReaderOptions
+{
+    AllowTrailingCommas = true,
+    CommentHandling = JsonCommentHandling.Skip
+};
+var reader = new Utf8JsonReader(jsonBytes, readerOptions);
+
+while (reader.Read())
+{
+    Console.Write(reader.TokenType);
+
+    switch (reader.TokenType)
+    {
+        case JsonTokenType.PropertyName:
+        case JsonTokenType.String:
+            {
+                string? text = reader.GetString();
+                Console.Write(" ");
+                Console.Write(text);
+                break;
+            }
+
+        case JsonTokenType.Number:
+            {
+                int intValue = reader.GetInt32();
+                Console.Write(" ");
+                Console.Write(intValue);
+                break;
+            }
+
+            // Other token types elided for brevity
+    }
+    Console.WriteLine();
+}
+// StartObject
+// PropertyName date
+// String 2023-08-02T09:00:00+00:00
+// PropertyName temp
+// Number 42
+// EndObject
+```
+
+## Key Features
+
+<!-- The key features of this package -->
+
+TODO
+
+## Main Types
+
+<!-- The main types provided in this library -->
+
+The main types provided by this library are:
+
+* `System.Text.Json.JsonSerializer`
+* `System.Text.Json.JsonDocument`
+* `System.Text.Json.JsonElement`
+* `System.Text.Json.Nodes.JsonNode`
+* `System.Text.Json.Utf8JsonWriter`
+* `System.Text.Json.Utf8JsonReader`
+
+## Addtional Documentation
+
+* [Conceptual documentation](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/overview)
+* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/system.text.json)
+
+## Related Packages
+
+<!-- The related packages associated with this package -->
+
+* Lightweight data formats abstraction: [System.Memory.Data](https://www.nuget.org/packages/System.Memory.Data/)
+* Serialization of HttpContent: [System.Net.Http.Json](https://www.nuget.org/packages/System.Net.Http.Json/)
+
+
+## Feedback & Contributing
 
-For more information, see the documentation:
```

Comment:
I recall that when we originally added this document we deliberately avoided inlining documentation content to avoid creating a second source of truth that would eventually go out of sync.

Reply:
Correct, so what's going here is mostly links and some simple samples.  Add any additional links you'd like, or reduce the samples if you feel there is too much duplication.